core/src/main/java/discord4j/core/spec/MessageCreateSpec.java

```diff
@@ -57,17 +56,154 @@
     private AllowedMentionsData allowedMentionsData;
     private MessageReferenceData messageReferenceData;
 
+
+    // Constants
+    public final static byte MARKDOWN_ITALIC = 0;
```

Comment:
Having byte constants isn't very idiomatic considering an enum would work just as well (a "style" enum would also have more intuitive code completion for the user).